server/amp/handlers/infinite-scroll.js

```diff
@@ -12,7 +12,7 @@ async function storyPageInfiniteScrollHandler(req, res, next, { client, config }
     client,
     queryParams: req.query,
   });
-  const jsonResponse = await infiniteScrollAmp.getResponse({ itemsTaken: 5 }); // itemsTaken has to match with itemsToTake in ampStoryPageHandler
+  const jsonResponse = await infiniteScrollAmp.getResponse();
```

Comment:
@VeenaYemmiganur will have to pass `infiniteScrollSource` to this class instance otherwise it breaks for `relatedStoriesApi`